components/add_users_to_team_modal/add_users_to_team_modal.test.tsx

```diff
@@ -53,7 +53,7 @@ describe('components/admin_console/add_users_to_team_modal/AddUsersToTeamModal',
 
         actions: {
             getProfilesNotInTeam: jest.fn(),
-            searchProfiles: jest.fn(),
+            searchProfiles: jest.fn().mockResolvedValue({data: []}),
```

Comment:
A lot of these were just actions with missing return values. The code using that return value would then error out asynchronously.